### ISSUE TITLE
No H1 heading is defined for the "Initial setup" Page: A11y_Maow_Initial Setup_Info and Relationships.

### ISSUE THREAD

**Issue by AnKushSingh05** [microsoft/moaw#124]
### GitHub Tags
#A11yTCS; #A11ySev3; #ChromiumEdge; #WCAG1.3.1; #Win11; #GH_Maow_Web_Mar24; #Maow; #Info and Relationships; #AILimited; #DesktopWeb; #A11yMAS;

### Actual:
No H1 heading is defined for the "Initial setup" Page

### Expected:
H1 heading should be defined for the "Initial setup" Page.
### 
### Environment Details:
Application Name: Maow
#URL: https://moaw.dev/workshop/create-workshop/?step=1
OS: Windows 11 Enterprise 23H2 (OS build 22631.3296)
Microsoft Edge: Version 122.0.2365.92 (Official build) (64-bit)

### Repro Steps:

1. Hit the URL https://moaw.dev/workshop/create-workshop/
2. Create A Workshop screen will get open.
3. TAB till "Initial setup" control present in left navigation and press ENTER.
4. TAB till "Initial setup" section.
5. Verify that H1 heading is defined for the "Initial setup" Page or not.

### User Impact:
User not able to understand the current structure of the page if no H1 heading provided for the page.

### Attachment:
![Bug_124](https://github.com/microsoft/moaw/assets/124244478/c6cfdfe3-3dd5-40f4-bb60-5e1eebff9109)

**Comment by AnKushSingh05** [comment]
> @AnKushSingh05 this one has been fixed, could you validate it?

Verified the issue in below environment
https://moaw.dev/
Issue is not fixed. Now 2 heading H1 and H2 is coming please remove the h2 heading and there only H1 should be fine.
Please reactivate the bug
![Screenshot (1956)](https://github.com/microsoft/moaw/assets/124244478/992c5473-b760-427c-b6d0-4f4cf106f8da)

**Comment by sinedied** [comment]
Thank you, the content hierarchy has been fixed.
@AnKushSingh05 could you validate it?

**Comment by AnKushSingh05** [comment]
Verified the issue in below environment.
https://moaw.dev/workshop/create-workshop/?step=1
Issue is fixed. Hence closing the bug.
![Screenshot (1972)](https://github.com/microsoft/moaw/assets/124244478/ca12f36e-b579-4726-8b76-b939c50d6afa)
